templates/macros.html

```diff
@@ -45,8 +45,12 @@
 
 {% macro composes(row) %}
 	{% if row.second_compose != None %}
-		<b>First: </b>{{row.compose}}<br><b>Second: </b>{{row.second_compose}}
+		<p style="text-align: center;"><b>From</b></p>
+			{{ row.compose }}
+		<hr />
+		<p style="text-align: center;"><b>To</b></p>
+			{{ row.second_compose }}
 	{% else %}
-		{{row.compose}}
+		{{ row.compose }}
```

Comment:
Curious why you made this change?

Reply:
It is just, because I am used to write it this way. It's already fixed.

Reply:
Generally good to follow existing convention in a project unless there is a reason to make a change. Can you make the whole thing consistent please?